src/csharp/Microsoft.Spark/SparkFiles.cs

```diff
@@ -2,33 +2,62 @@
 // The .NET Foundation licenses this file to you under the MIT license.
 // See the LICENSE file in the project root for more information.
 
+using System;
+using System.IO;
 using Microsoft.Spark.Interop;
 using Microsoft.Spark.Interop.Ipc;
 
 namespace Microsoft.Spark
 {
     /// <summary>
-    /// Resolves paths to files added through `SparkContext.addFile()`.
+    /// Resolves paths to files added through
+    /// <c>SparkContext.addFile()</c>.
     /// </summary>
     public static class SparkFiles
     {
         private static IJvmBridge Jvm { get; } = SparkEnvironment.JvmBridge;
         private static readonly string s_sparkFilesClassName = "org.apache.spark.SparkFiles";
 
+        [ThreadStatic]
+        private static string s_rootDirectory;
+
+        [ThreadStatic]
+        private static bool s_isWorker;
+
         /// <summary>
-        /// Get the absolute path of a file added through `SparkContext.addFile()`.
+        /// Get the absolute path of a file added through
+        /// <c>SparkContext.addFile()</c>.
```

Comment:
any reason this is on a new line?

Reply:
same in other places.

Reply:
no significant reason, Just hit max line limit and thought it may look nicer if it wasn't too long.  Reverted.